pkg/support/support.go

```diff
@@ -320,6 +328,16 @@ func getFinchVersion() string {
 	return version.Version
 }
 
+func (bb *bundleBuilder) getFinchVersion() string {
+	cmd := bb.ecc.Create("finch", "version")
```

Comment:
is it worth replacing "finch" with `os.Executable()`?

Reply:
Sure, I can change this. I have to fix a few lint failures as well. Sending a new revision with all the changes.

Reply:
Cool. You can use the `ExecutableFinder` in `pkg/system` to make mocking easier. There's some other usages in the code base